Assignment2/Assignment2_QNo2/SolAssignmentNo2_QNo2.rb

```diff
@@ -0,0 +1,51 @@
+# => 2. Write a class (with constructor) to define instance
+# => methods for getting area & perimeter of a rectangle.
+
+# => Business logic class
+class RectangleSkeleton
+
+  attr_accessor :len, :br
```

Comment:
@HrisikeshSinghaChowdhury Here should not be any line gap between block start and first statement in that block